src/menu/menu-item.mapper.ts

```diff
@@ -28,6 +28,7 @@ export class MenuItemMapper {
           id: modificationGroup.modificationGroup.id,
           isMultipleChoice:
             modificationGroup.modificationGroup.isMultipleChoice,
+          isMandatory: modificationGroup.modificationGroup.isMandatory,
```

Comment:
also, why do we have so deeply nested object? Hard to understand why there is modificationGroup inside modificationGroup. `modificationGroup.modificationGroup.modifications`